stable/sonarqube/README.md

```diff
@@ -52,6 +52,6 @@ The following table lists the configurable parameters of the Sonarqube chart and
 | `postgresql.postgresPassword`               | Postgresql database password        | `sonarPass`                                |
 | `postgresql.postgresDatabase`               | Postgresql database name            | `sonarDB`                                  |
 
-You can also configure values for the PostgreSQL database via the Postgresql [README.md](https://github.com/kubernetes/charts/blob/master/stable/postgresql/README.md)
+You can also configure values for the PostgreSQL database via the Postgresql [README.md].(https://github.com/kubernetes/charts/blob/master/stable/postgresql/README.md)
```

Comment:
Please remove the period, it in is the middle of a hyperlink

Reply:
OK, It's done.